helm-chart/ai-service-chart/Chart.yaml

```
@@ -0,0 +1,8 @@
+name: ai-service-module-chart 
```

YAMLlint (1.37.1)

[error] trailing spaces

(trailing-spaces)